exceptional_operations.md

```diff
@@ -5,36 +5,37 @@ css_id: exceptional_operations
 
-In the second option, an OEM MUST increment the release counter of an image whenever it is critical that an ECU not install images with lower release counters. In the example in Figure 3, if an ECU installs “foo.img,” then it cannot install “bar.img.” This is done to prevent the installation of compatible images with lower release counters that have known security vulnerabilities, rather newer images in which these vulnerabilities have been fixed.
+In the second option, an OEM increments the release counter of an image whenever it is critical that an ECU not install images with lower release counters. In the example in Figure 3, if an ECU installs “foo.img,” then it cannot install “bar.img.” This is done to prevent the installation of compatible images with lower release counters that have known security vulnerabilities, rather than newer images in which these vulnerabilities have been fixed.
 
-### TODO
-Insert figure except_2_diffimage_samecounterprev2.jpg and caption
+<img align="center" src="assets/images/except_3_diffimage_samecounter2.png" width="500" style="margin: 0px 20px"/>
+
+**Figure 3.** *Uptane forbids the installation of images with lower release counters than what is currently installed.*
 
 The advantage to this method is that it prevents rollback attacks in a situation
-where attackers compromise only the director repository.  However, there are two disadvantages. First, the release counters for images have to be maintained, even if role B now signs for images previously signed by role A. This is because release counters are always compared to previous targets metadata files. Second, it is more cumbersome to roll back updates, or deliberately cause ECUs to install older images, because offline keys are used to increment the release counters of these older images in the new targets metadata for the image repository. However, this method SHOULD be preferred, because it is more secure. See the Security pages on this website for more techniques that can be used to limit rollback attacks when the director repository is compromised.
+where attackers compromise only the Director repository.  However, there are two disadvantages. First, the release counters for images have to be maintained, even if role B now signs for images previously signed by role A. This is because release counters are always compared to previous Targets metadata files. Second, it is more cumbersome to roll back updates, or deliberately cause ECUs to install older images, because the release counters of these older images are incremented in the new Targets metadata for the Image repository with offline keys. However, this method SHOULD be preferred, because it is more secure. See the [Security pages](https://github.com/uptane/deployment-considerations/blob/master/security_considerations.md) on this website for more techniques that can be used to limit rollback attacks when the Director repository is compromised.
```

Comment:
```suggestion
where attackers compromise only the Director repository.  However, there are two disadvantages. First, the release counters for images have to be maintained, even if role B now signs for images previously signed by role A. This is because release counters are always compared to previous Targets metadata files. Second, it is more cumbersome to roll back updates, or deliberately cause ECUs to install older images, because the release counters of these older images are incremented in the new Targets metadata for the Image repository with offline keys. However, this method SHOULD be preferred, because it is more secure. See the [security considerations](https://github.com/uptane/deployment-considerations/blob/master/security_considerations.md) document for more techniques that can be used to limit rollback attacks when the Director repository is compromised.
```